src/rp2040/hardware_regs/rp2040.svd

```diff
@@ -22331,7 +22331,7 @@
             </field>
           </fields>
           <name>STARTUP</name>
-          <resetValue>0x00000000</resetValue>
+          <resetValue>0x000000c4</resetValue>
```

Comment:
note i saw this in passing, which was I think missed from the previous change #478

Reply:
This SVD file is auto-generated, so shouldn't be hand-edited as any changes will just get overwritten. I'll do my best to propagate the changes you've made here into the "upstream source files".

Reply:
it looks like this must have been hand edited b4 otherwise it wouldn't be out of step

Reply:
Where is it "out of step"? It says `<resetValue>0x00000000</resetValue>` here because https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2040/hardware_regs/include/hardware/regs/xosc.h#L142 says `#define XOSC_STARTUP_DELAY_RESET  "-"` (i.e. unknown / undefined).
https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2040/hardware_regs/include/hardware/regs/xosc.h and https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2040/hardware_regs/rp2040.svd are both generated from the same "upstream source" :slightly_smiling_face:

Reply:
Yeah the correct reset value is 0xc4 but there is currently no way to express the actual as-taped-out reset value of an `undef` field in VRB, so I added it to the info string so that it at least appears in the datasheet and header. I'll go see if there is an internal issue for this already.

Reply:
Fixed in #601